docs/md/flows.md

```diff
@@ -353,7 +353,7 @@ through `extra` on the result.
                     "category": "",
                     "category_localized": "",
                     "created_on": "2018-04-11T18:24:30.123456Z",
-                    "input": null,
+                    "input": "",
```

Comment:
ditching the whole differentiation between `input == null` and `input == ""`.. we don't make that distinction else where and this makes a lot of code simpler